tasks.py

```diff
@@ -29,7 +29,7 @@
         "ghuser": {
             "source_dir": "src/compas_timber/ghpython/components",
             "target_dir": "src/compas_timber/ghpython/components/ghuser",
-            "prefix": "(COMPAS_TIMBER)",
+            "prefix": "CT_",
```

Comment:
I started using `Projectname: ` in core and fab components, maybe change to `CT: `?